src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/interior_data_manager_impl.cc

```diff
@@ -126,8 +126,8 @@ void InteriorDataManagerImpl::UpdateHMISubscriptionsOnAppUnregistered(
 void InteriorDataManagerImpl::UnsubscribeFromInteriorVehicleData(
     const ModuleUid& module) {
   cache_.Remove(module);
-  auto unsubscribe_request = RCHelpers::CreateUnsubscribeRequestToHMI(
-      module, app_mngr_.GetNextHMICorrelationID());
+  auto unsubscribe_request = RCHelpers::CreateGetInteriorVDRequestToHMI(
+      module, app_mngr_.GetNextHMICorrelationID(), false);
```

Comment:
propose to use small enum defined near the CreateGetInteriorVDRequestToHMI.
enum GetInteriorData {
  SUBSCRIBE,
  UNSUBSCRIBE
}
and then `CreateGetInteriorVDRequestToHMI(module, app_mngr_.GetNextHMICorrelationID, GetInteriorData::UNSUBSCRIBE)`

Reply:
@LuxoftAKutsan 
Done in:https://github.com/LuxoftSDL/sdl_core/pull/99/commits/096290046381d623d3244d8d3b265f8639f0ca7a